scripts/build-brotli.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
```

Comment:
can you also revert this? Sorry for being so picky here. I just dont want something to break lol

Reply:
Ahhh my bad! I thought I was actually bringing this change back from upstream, I'm not sure how I handled this, sorry

Reply:
wait actually, I restored what was on upstream: `#!/bin/bash`

Reply:
okay nice!